server/game/cards/09.4-TCoH/TogashiYoshi.js

```diff
@@ -0,0 +1,31 @@
+import { TargetModes } from '../../Constants.js';
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class TogashiYoshi extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Gain 1 fate of an unclaimed ring',
+            when: {
+                afterConflict: (event, context) => event.conflict.winner === context.player &&
+                    context.source.isParticipating()
+            },
+            target: {
+                mode: TargetModes.Ring,
+                ringCondition:  ring => ring.fate >= 1 &&
+                    ring.contested === false &&
+                    ring.claimed === false
+            },
```

Comment:
This is targeting when there is no choose in the action, this could cause some issues with Hantei 38.  I think we should be using a SelectRingAction.